src/components/MapView/utils.ts

```diff
@@ -46,3 +49,83 @@ export const getFeatureInfoParams = (
 
   return params;
 };
+
+export const convertToTableData = (
+  result: ExposedPopulationResult,
+  groupBy: string,
+) => {
+  const { key, statistic } = result;
+  const { features } = result.featureCollection;
+  const fields = _.uniq(
+    _.map(features, f => f.properties && f.properties[key]),
```

Comment:
optional chaining?

Reply:
Yeah. New to Typescript, I am loving it :) Check correction below

```typescript
const { key, statistic, featureCollection: { features } } = result;
```